provision-enterprise-subsidy.sh

```diff
@@ -8,23 +8,23 @@ docker-compose up -d --build
 
 # Wait for MySQL
 echo "Waiting for MySQL"
-until docker exec -i enterprise_subsidy.db mysql -u root -se "SELECT EXISTS(SELECT 1 FROM mysql.user WHERE user = 'root')" &> /dev/null
+until docker exec -i enterprise-subsidy.db mysql -u root -se "SELECT EXISTS(SELECT 1 FROM mysql.user WHERE user = 'root')" &> /dev/null
 do
   printf "."
   sleep 1
 done
 sleep 5
 
 # Create the database
-docker exec -i enterprise_subsidy.db mysql -u root -se "CREATE DATABASE enterprise_subsidy;"
+docker exec -i enterprise-subsidy.db mysql -u root -se "CREATE DATABASE enterprise-subsidy;"
```

Comment:
I don't think updating the actual mysql database name here is necessary, but I did it for consistency. LMK what you think.